### ISSUE TITLE
yarn build & yarn start failing on initial project

### ISSUE THREAD

**Issue by bayswaterpc** [pankod/superplate#282]
I tried running on a freshly generated project and I found that they were failing on the  typescript compilation of the examples folder
![image](https://user-images.githubusercontent.com/78702667/133264886-60c46439-ea88-4a3f-841a-30a5aa1d8ca2.png)

I followed the error prompts and adjusted the files so they would work.  The fixes can be seen on this branch of my ongoing project for those who are curious.

https://github.com/bayswaterpc/bayswaterpc-superplate-blog/tree/FixBuildYarnStartErrors

**Comment by akinwol** [comment]
@bayswaterpc  agreed. I really like this boilerplate but have to do a clean up every time or just get rid of the examples

**Comment by omeraplak** [comment]
> @bayswaterpc agreed. I really like this boilerplate but have to do a clean up every time or just get rid of the examples

Hey @akinwol ,
Would you consider creating a PR for the parts you fixed?
